packages/sdk/src/contracts/holograph.contract.ts

```diff
@@ -6,6 +6,11 @@ import {Config} from '../services/config.service'
 import {HolographABI} from '../constants/abi/develop'
 import {HolographByNetworksResponse, getContract, getSelectedNetworks, mapReturnType} from '../utils/contracts'
 import {HolographLogger} from '../services/logger.service'
+import {normalizeException} from '../errors/error-info'
+import {isCallException} from 'ethers'
```

Comment:
move this import to the top, to keep the pattern of package imports first, then file imports